### DISCUSSION TITLE
orElse(...). ?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [gnat/surreal#32] [Q&A]
I wish I could do this:
```javascript
me('some css selector').run(...element found, do something....).orElse(... do something if element is not found ...);
```

**Comment by gnat** [comment]
Really interesting idea! Will look into how this interacts with the rest of surreal and consider adding it.

**Comment by hrstoyanov** [comment]
Thanks! Did you already add it, or still thinking about it?
